client/my-sites/subscribers/components/empty-list-view/styles.scss

```diff
@@ -0,0 +1,84 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.empty-list-view {
+	color: $studio-gray-100;
+	font-family: "SF Pro Text", $sans;
+	margin-top: 92px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+
+	.empty-list-view__title {
+		font-size: $font-title-small;
+		font-weight: 500;
+		line-height: rem(26px);
+		margin-bottom: 8px;
+		letter-spacing: 0.38px;
+		text-align: center;
+	}
+
+	.empty-list-view__description {
+		font-size: $font-body;
+		font-weight: 400;
+		line-height: rem(24px);
+		margin-bottom: 24px;
+		text-align: center;
+	}
+
+	.empty-list-view__cta-link {
+		width: 571px;
+		height: 52px;
+		margin-bottom: 12px;
+		box-shadow: 0 0 1px $studio-gray-5;
+		border-radius: 4px;
```

Comment:
```suggestion
		border: 1px solid $studio-gray-5;
		border-radius: 3px;
		box-shadow: none;
```

Reply:
Doing `border-radius: 3px;` adds a lint issue, as it's only allowed 2 or 4. That's why I moved to 4px.

Reply:
Interesting! Good to know. Thanks for sharing.